package.json

```diff
@@ -53,7 +53,7 @@
     "jest": "^20.0.4"
   },
   "dependencies": {
-    "ember-rfc176-data": "^0.2.7",
+    "ember-rfc176-data": "~0.3.0",
```

Comment:
Should probably still be `^0.3.0` (it just has the same effect as `~0.3.0` in this case)